tests/console/suseconnect.pm

```diff
@@ -0,0 +1,44 @@
+# Summary: 
+# Maintainer: Juraj Hura <jhura@suse.com>
+
+use base "basetest";
+use strict;
+use testapi;
+use utils 'zypper_call';
+use registration;
+use version_utils 'is_sle';
+
+sub run {
+    my $reg_code = get_required_var("SCC_REGCODE");
+    my $arch = get_required_var("ARCH");
+    my $live_reg_code = get_required_var("SCC_REGCODE_LIVE");
+
+    select_console 'root-console';
+    
+    # Make sure to start with de-registered system. In case the system is not registered this command will fail
+    assert_script_run "SUSEConnect -d ; echo 1";
+    assert_script_run "SUSEConnect --cleanup";
+    assert_script_run "SUSEConnect --status-text";
+    
+    zypper_call 'lr';
+    zypper_call 'services';
+    zypper_call 'products';
+
+    assert_script_run "SUSEConnect -r $reg_code";
+    assert_script_run "SUSEConnect --status-text| grep -v 'Not Registered'";
+    zypper_call 'ref';
+    assert_script_run "SUSEConnect --list-extensions";
+
+    if (is_sle('15+')) {
+        add_suseconnect_product('sle-module-live-patching', undef, undef, "-r $live_reg_code");
+    }
+    elsif (is_sle('<15')) {
+        add_suseconnect_product('sle-live-patching', undef, undef, "-r $live_reg_code");
+    }
```

Comment:
how about

```suggestion
    add_suseconnect_product(is_sle('<15') ? 'sle-live-patching' : 'sle-module-live-patching', undef, undef, "-r $live_reg_code");
```

I wonder if that module is a good choice. Do we have it available also for the products in development? Also I don't think the if/elsif is a good approach, better just use if/else (or what I suggested). Would not make sense to call the whole test module when neither is true, right?

Reply:
Thx for that avoidance of if. As for development, that will be problem for whoever wants to use it there.